pkg/repository/db.go

```diff
@@ -14,7 +14,8 @@ import (
 type GroceryDB interface {
 	getGroceryByID(string) (models.Grocery, error)
 	getGrocerys() ([]models.Grocery, error)
-	GetGrocerysByQuantity(int) ([]models.Grocery, error)
+	addGrocery([]models.Grocery) error
```

Comment:
This is not the same as what the function actually does. The function accepts a single grocery but the interface says it accepts a slice.

Really maybe we shouldn't even be having this interface yet since it isn't used.